scapy/contrib/isotp.py

```diff
@@ -465,22 +466,26 @@ def __init__(self):
 
     @staticmethod
     def sniff(opened_socket, count=0, store=True, timeout=None,
-              prn=None, stop_filter=None, lfilter=None, started_callback=None):
+              prn=None, stop_filter=None, lfilter=None, started_callback=None,
+              use_ext_addr=None):
         from scapy import plist
-        m = ISOTPMessageBuilder()
+        m = ISOTPMessageBuilder(use_ext_addr=use_ext_addr)
         c = ISOTPSniffer.Closure()
         c.max_count = count
 
         def internal_prn(p):
+            print("Feeding ", repr(p))
             m.feed(p)
             while not c.stop and m.count() > 0:
                 rcvd = m.pop()
+                print("Popped ", repr(rcvd))
                 on_pkt(rcvd)
 
         def internal_stop_filter(p):
             return c.stop
 
         def on_pkt(p):
+            print(repr(p))
```

Comment:
Apart from that those are probably debug comments

Reply:
Fixed